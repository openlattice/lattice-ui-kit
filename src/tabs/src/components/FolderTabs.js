import Tabs from '@material-ui/core/Tabs';
import { withStyles } from '@material-ui/core/styles';

import { NEUTRAL } from '../../../colors';

const FolderTabs = withStyles({
  root: {
    borderBottom: `1px solid ${NEUTRAL.N100}`,
    minHeight: '32px',
  },
  indicator: {
    backgroundColor: 'transparent',
  },
  flexContainer: {
    margin: '2px 8px 0',
  },
})(Tabs);

export default FolderTabs;
