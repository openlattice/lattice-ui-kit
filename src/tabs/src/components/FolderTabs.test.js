import toJson from 'enzyme-to-json';
import { mount } from 'enzyme';

import FolderTabs from './FolderTabs';

describe('FolderTabs', () => {

  describe('default', () => {
    test('should match snapshot', () => {
      const wrapper = mount(<FolderTabs />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
