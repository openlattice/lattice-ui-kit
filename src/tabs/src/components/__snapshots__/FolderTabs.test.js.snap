// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FolderTabs default should match snapshot 1`] = `
<WithStyles(WithStyles(ForwardRef(Tabs)))>
  <WithStyles(ForwardRef(Tabs))
    classes={
      Object {
        "flexContainer": "WithStyles(ForwardRef(Tabs))-flexContainer-3",
        "indicator": "WithStyles(ForwardRef(Tabs))-indicator-2",
        "root": "WithStyles(ForwardRef(Tabs))-root-1",
      }
    }
  >
    <ForwardRef(Tabs)
      classes={
        Object {
          "centered": "MuiTabs-centered",
          "fixed": "MuiTabs-fixed",
          "flexContainer": "MuiTabs-flexContainer WithStyles(ForwardRef(Tabs))-flexContainer-3",
          "flexContainerVertical": "MuiTabs-flexContainerVertical",
          "indicator": "MuiTabs-indicator WithStyles(ForwardRef(Tabs))-indicator-2",
          "root": "MuiTabs-root WithStyles(ForwardRef(Tabs))-root-1",
          "scrollButtons": "MuiTabs-scrollButtons",
          "scrollButtonsDesktop": "MuiTabs-scrollButtonsDesktop",
          "scrollable": "MuiTabs-scrollable",
          "scroller": "MuiTabs-scroller",
          "vertical": "MuiTabs-vertical",
        }
      }
    >
      <div
        className="MuiTabs-root WithStyles(ForwardRef(Tabs))-root-1"
      >
        <div
          className="MuiTabs-scroller MuiTabs-fixed"
          onScroll={[Function]}
          style={
            Object {
              "marginBottom": null,
              "overflow": "hidden",
            }
          }
        >
          <div
            className="MuiTabs-flexContainer WithStyles(ForwardRef(Tabs))-flexContainer-3"
            onKeyDown={[Function]}
            role="tablist"
          />
          <WithStyles(ForwardRef(TabIndicator))
            className="MuiTabs-indicator WithStyles(ForwardRef(Tabs))-indicator-2"
            color="secondary"
            orientation="horizontal"
            style={
              Object {
                "left": 0,
                "width": 0,
              }
            }
          >
            <ForwardRef(TabIndicator)
              className="MuiTabs-indicator WithStyles(ForwardRef(Tabs))-indicator-2"
              classes={
                Object {
                  "colorPrimary": "PrivateTabIndicator-colorPrimary-5",
                  "colorSecondary": "PrivateTabIndicator-colorSecondary-6",
                  "root": "PrivateTabIndicator-root-4",
                  "vertical": "PrivateTabIndicator-vertical-7",
                }
              }
              color="secondary"
              orientation="horizontal"
              style={
                Object {
                  "left": 0,
                  "width": 0,
                }
              }
            >
              <span
                className="PrivateTabIndicator-root-4 PrivateTabIndicator-colorSecondary-6 MuiTabs-indicator WithStyles(ForwardRef(Tabs))-indicator-2"
                style={
                  Object {
                    "left": 0,
                    "width": 0,
                  }
                }
              />
            </ForwardRef(TabIndicator)>
          </WithStyles(ForwardRef(TabIndicator))>
        </div>
      </div>
    </ForwardRef(Tabs)>
  </WithStyles(ForwardRef(Tabs))>
</WithStyles(WithStyles(ForwardRef(Tabs)))>
`;
